src/components/MessageBar/MessageBar.jsx

```diff
@@ -10,35 +19,138 @@ import {
   SendIcon,
 } from './MessageBarStyled';
 
-const MessageBar = () => {
-  const [value, setValue] = useState('');
+const MessageBar = ({
+  countryData,
+  setCountryData,
+  setSubscriptionCountryRooms,
+}) => {
+  const [message, setMessage] = useState('');
+  const [isShowJoinBtn, setIsShowJoinBtn] = useState(true);
+  const userId = useSelector(getUser)?.id;
   const textAreaRef = useRef(null);
```

Comment:
Why do you need this ref?

Reply:
I don't know. I think it`s leave from time when I tried to make custom textarea autosize.
I removed it.
Done